src/Http/WebUtilities/src/MultipartSectionStreamExtensions.cs

```diff
@@ -17,18 +17,24 @@ public static class MultipartSectionStreamExtensions
     /// <param name="section">The section to read from</param>
     /// <returns>The body steam as string</returns>
     public static async Task<string> ReadAsStringAsync(this MultipartSection section)
+        => await section.ReadAsStringAsync(CancellationToken.None);
```

Comment:
```suggestion
    public static Task<string> ReadAsStringAsync(this MultipartSection section)
        => section.ReadAsStringAsync(CancellationToken.None);
```

Reply:
I preserved the await because the method it calls also doesn't just return the raw `Task`.

Reply:
With the refactor to remove the block, shall I remove both then if/when the API changes are approved?

Reply:
The other method still needs the await so the using statement doesn't dispose the reader too soon. Please remove the await from this overload though.